roles/linux_ansible_init/files/ansible-init.py

```diff
@@ -45,6 +60,12 @@ def ansible_exec(cmd, *args, **kwargs):
 user_metadata = response.json().get("meta", {})
 
 
+logger.info("checking for ansible-init disable flag")
+if user_metadata.get("ansible_init_disable", "false").lower() == "true":
+    logger.info("ansible-init is disabled via metadata. Exiting.")
+    exit(0)  # Exit early if ansible-init is disabled.
```

Comment:
This should be `sys.exit`.

`exit` is intended mainly for use in the REPL - see https://docs.python.org/3/library/constants.html#exit